### DISCUSSION TITLE
警告：更新前一定要仔细阅读更新日志！！！

### DISCUSSION THREAD

**Discussion by japengwu** [the1812/Bilibili-Evolved#2686] [综合讨论 / General]
本来以为只是一次普通的更新，没想到变化这么大，再加上我更新出现了问题，不知道为什么用不了。
所以有点懵，说话有点冲动。
这里我先道个歉。
---------------------------------------以下是原内容-----------------------------------------------
更新之后所有的功能都没了。啥功能都没了！啥功能都没了！！所有的功能！全部！都没了！！！
而且变不回原来的版本了。
我真是手欠，没事更新它干嘛。

更新之后，首页简化没有了。多了个功能栏，里面空空如也。
多了模组、插件、样式三个功能，这三个功能全都是空架子，用不了。这三个功能都有在线仓库，但是在线仓库里面全都是空的。

其他的全都是空的。

老版本没有备份，现在回不去了。简直太难受了。

**Comment by Geeyun-JY3** [comment]
能不能先看看更新日志：https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.0.7

想要老版本也回得去：https://github.com/the1812/Bilibili-Evolved/discussions/2640#discussioncomment-1789982

**Reply by japengwu** [reply]
非常感谢！改回来了
我以为只是普通的小更新，没想到区别这么大。

不过更新完了之后确实啥都没有了，更新日志上的那些功能，一个都没有，整个都是空的。“功能”和“设置”里面都是空的。
在线仓库也是空的。

现在想想，是不是因为我没有科学上网？

**Reply by Geeyun-JY3** [reply]
我还是建议你再认真地看一次更新日志，善用一下搜索。功能肯定是有的。网络条件不一定是原因，理论上 jsDelivr 应该是没问题的。
之前也有的是报告在线仓库用不了的：#2560、#2614、#2650、#2664

而且你什么信息都没有，我也很难帮你判断。

**Reply by japengwu** [reply]
OK谢谢，哪天闲下来我研究研究

**Reply by FoundTheWOUT** [reply]
唱黑脸就靠你了🤪

**Reply by japengwu** [reply]
> 
我那天有点懵，说话也有点冲动。。。其实也一直想道歉来着

**Comment by xuwenchao980507** [comment]
我是因为组件在线仓库搜不到v1设置，没法添加我旧版设置所以用不了，油猴版本是beta 4.14.6148 。我还是老老实实用v1吧，前两天就已经回退回来了

**Reply by the1812** [reply]
那是自然, 现在只有非 beta 版油猴能用(

**Reply by FoundTheWOUT** [reply]
感觉怎么挺多用 beta油猴的，是安装问题吗

**Reply by Geeyun-JY3** [reply]
「一定要仔细阅读更新日志！！！」

**Reply by the1812** [reply]
现在 beta 好像有 6149 版了, 不知道修复了没

**Reply by xuwenchao980507** [reply]
> 现在 beta 好像有 6149 版了, 不知道修复了没

商店里没有新版咱也下载不了，不过我黑猴红猴都有，刚试了下黑猴没问题，只是我红猴几十个脚本都得再装一遍

**Reply by xuwenchao980507** [reply]
> 现在 beta 好像有 6149 版了, 不知道修复了没

感谢大大

**Reply by xuwenchao980507** [reply]
> 感觉怎么挺多用 beta油猴的，是安装问题吗

倒也不是，因为黑猴用不了B站番剧解锁只能用红猴，黑猴我仓库里倒是一直备着

**Reply by xuwenchao980507** [reply]
> 「一定要仔细阅读更新日志！！！」

咳咳，感谢！可能我点进来只看到怎么设置的教程了吧哈哈

**Reply by Geeyun-JY3** [reply]
> 现在 beta 好像有 6149 版了, 不知道修复了没

今天终于收到推送了。
测试 6149 版已正常。
